src/utils/i18n.ts

```diff
@@ -178,7 +178,7 @@ export function lookupLocale(
     // Only add them if they aren't subsets of the next locale or if they appear later in the
     // preference list.
     const next = normalizedPreferredlocales[i + 1] || '';
-    const subsets = makeSubsets(current).filter((s) => !baseSet[s] && next.search(s) === -1);
+    const subsets = getFallbackPattern(current).filter((s) => !baseSet[s] && next.search(s) === -1);
```

Comment:
The new inclusive behavior of `getFallbackPattern` doesn't affect this logic, right? I don't really understand what this is doing, to be honest...

Reply:
It adds a trivial amount of redundancy to this method.